src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -137,6 +141,15 @@ const MintTokenMotion = ({
           />
         </div>
         <div className={styles.details}>
+          {motionState === MotionState.StakeRequired && (
+            <StakingWidget
+              motionId={motionId}
+              motionDomainId={motionDomainId}
+              rootHash={rootHash}
+              colonyAddress={colony.colonyAddress}
+              tokenDecimals={decimals}
+            />
+          )}
```

Comment:
I don't think this is correct, since if there is "some", but not 100% stake, you basically won't show the widget

Reply:
What would be a better check? In the design, the widget to `Stake` only seems to appear when the Motion is in a `Stake required` status

Reply:
This should show for these three states:
- MotionState.StakeRequired
- MotionState.Motion
- MotionState.Objection